pkg/skaffold/kaniko/kaniko.go

```diff
@@ -54,10 +54,11 @@ func RunKanikoBuild(ctx context.Context, out io.Writer, artifact *v1alpha2.Artif
 		return "", errors.Wrap(err, "starting log streamer")
 	}
 	imageDst := fmt.Sprintf("%s:%s", artifact.ImageName, initialTag)
-	p, err := client.CoreV1().Pods("default").Create(&v1.Pod{
+	p, err := client.CoreV1().Pods(cfg.Namespace).Create(&v1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   "kaniko",
-			Labels: map[string]string{"kaniko": "kaniko"},
+			Name:      "kaniko",
+			Labels:    map[string]string{"skaffold-kaniko": "skaffold-kaniko"},
```

Comment:
can we add some more information about how skaffold invoked kaniko? basically the relevant parts of the kaniko skaffold config

Reply:
I don't think there's actually anything else in there worth including. Any suggestions?

```
type KanikoBuild struct {
	GCSBucket      string `yaml:"gcsBucket,omitempty"`
	PullSecret     string `yaml:"pullSecret,omitempty"`
	PullSecretName string `yaml:"pullSecretName,omitempty"`
	Namespace      string `yaml:"namespace,omitempty"`
}
```

Reply:
why not all of it? should be useful for debugging if it doesn't invoke what they'd like

Reply:
SGTM